proto/nft/v1/tx.proto

```diff
@@ -45,11 +45,8 @@ message MsgTransferNFT {
 
     string id = 1;
     string denom_id = 2 [ (gogoproto.moretags) = "yaml:\"denom_id\"" ];
-    string name = 3;
-    string uri = 4 [ (gogoproto.customname) = "URI" ];
-    string data = 5;
-    string sender = 6;
-    string recipient = 7;
+    string sender = 3;
+    string recipient = 4;
```

Comment:
not a protobuf expert -- if this was meant to be still somewhat compatible with the original IRISnet module... what's the best way when the fields are removed?

Reply:
(if possible, not a major blocker)

Reply:
not changing the index?

Reply:
with protobuf rules, if you want to keep compatibility, never remove fields

Reply:
maybe no need -- the end goal will be to migrate to standard Cosmos SDK module... for compat with IRISnet -- the response data didn't change, so browsers/wallets should be able to query / view the same things?